src/Features/CSharp/Portable/InvertIf/CSharpInvertIfCodeRefactoringProvider.cs

```diff
@@ -69,6 +69,7 @@ protected override bool CanControlFlowOut(SyntaxNode node)
                 case SyntaxKind.SwitchSection:
                 case SyntaxKind.LocalFunctionStatement:
                 case SyntaxKind.SetAccessorDeclaration:
+                case SyntaxKind.InitAccessorDeclaration:
                 case SyntaxKind.GetAccessorDeclaration:
                 case SyntaxKind.AddAccessorDeclaration:
                 case SyntaxKind.RemoveAccessorDeclaration:
```

Comment:
```cs
            switch (node)
            {
                case SwitchSectionSyntax:
                case LocalFunctionStatementSyntax:
                case AccessorDeclarationSyntax:
                case MemberDeclarationSyntax:
                case AnonymousFunctionExpressionSyntax:
                case CommonForEachStatementSyntax:
                case DoStatementSyntax:
                case WhileStatementSyntax:
                case ForStatementSyntax:
                    return false;
            }
```

Reply:
Depending on the actual node type seems better than SyntaxKind.
@sharwell What do you think?

Reply:
yes, just depend on the actual node type.